vsgenstubs/_vapoursynth.part.pyi

```diff
@@ -360,22 +361,23 @@ class VideoNode:
 #include <plugins/bound>
 
     format: typing.Optional[Format]
-    
+
     fps: fractions.Fraction
     fps_den: int
     fps_num: int
-        
+
     height: int
     width: int
-    
+
     num_frames: int
+    flags: int
 
     def get_frame(self, n: int) -> VideoFrame: ...
     def get_frame_async_raw(self, n: int, cb: _Future[VideoFrame], future_wrapper: typing.Optional[typing.Callable[..., None]]=None) -> _Future[VideoFrame]: ...
     def get_frame_async(self, n: int) -> _Future[VideoFrame]: ...
 
-    def set_output(self, index: int=0, alpha: typing.Optional['VideoNode']=None) -> None: ...
-    def output(self, fileobj: typing.BinaryIO, y4m: bool = False, progress_update: typing.Optional[typing.Callable[[int], int]]=None, prefetch: int = 0) -> None: ...
+    def set_output(self, index: int=0, alpha: typing.Optional['VideoNode']=None) -> typing.NoReturn: ...
```

Comment:
As with some of the other functions, these do in fact return when called from a standard script.

Reply:
See https://github.com/vapoursynth/vsrepo/pull/125#discussion_r435502644, I removed the commit.